src/components/MoneyReportHeader.tsx

```diff
@@ -269,7 +281,7 @@ function MoneyReportHeader({session, policy, chatReport, nextStep, report: money
                     nonHeldAmount={!ReportUtils.hasOnlyHeldExpenses(moneyRequestReport.reportID) ? nonHeldAmount : undefined}
                     requestType={requestType}
                     fullAmount={fullAmount}
-                    isSmallScreenWidth={isSmallScreenWidth}
+                    isSmallScreenWidth={shouldUseNarrowLayout}
```

Comment:
Passing the `shouldUseNarrowLayout` value as `isSmallScreenWidth` will stretch the confirmation modal in the web. We should not have changed the `isSmallScreenWidth` value, as this change caused the issue https://github.com/Expensify/App/issues/45002